src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * <p>
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package org.fundacion.jala.converter.service.videoclasses;
+
+public class Converter {
+    private String startFirstCommand = "ffmpeg -i ";
+    private VideoParameter parameter;
+    private String format;
+    private String output;
+    private String pathOutput;
+    private static final int WAIT_TIME = 5000;
+    private static final int INIT_NUMBER = 20;
+
+    public Converter(final VideoParameter vParameter) {
```

Comment:
The name of the parameter should be more descriptive

Reply:
I agree with you Edson